### PR TITLE
Add kawaii mode

### PR DESCRIPTION
**Pull request by mozzius**
Adds kawaii mode. Activate by adding `?kawaii=true` and deactivate by adding `?kawaii=false` - kawaii mode is persisted in local storage. Web only

Kawaii mode swaps the logo and adds attribution

There are some minor style tweaks that should only apply to kawaii mode - please make sure that that is the case

### CODE REVIEW

**Review by pfrazee — APPROVED** [review]
Havent tested but code review lg

**Comment by gaearon** [comment]
What about `?uwu=true`? Or is that a convention. React site does `uwu`

**Comment by mozzius** [comment]
Hono went with `kawaii`, we could do `uwu` but kawaii is just what Emily suggested

**Review by haileyok — APPROVED** [review]
Glad we finally got around to doing this, it's been needed for a while!

**Comment by haileyok** [comment]
Tested on renderio, looks good.

File: src/view/shell/Drawer.tsx [hunk 1]
```diff
@@ -262,6 +264,17 @@ let DrawerContent = ({}: {}): React.ReactNode => {
               href="https://bsky.social/about/support/privacy-policy"
               text={_(msg`Privacy Policy`)}
             />
+            {kawaii && (
+              <Text type="md" style={pal.textLight}>
+                Logo by{' '}
```
Comment (surfdude29) [inline src/view/shell/Drawer.tsx:269]: Sorry to butt in, what about wrapping this section in `<Trans>` so that it can be translated into Japanese?

**Comment by heikadog** [comment]
holy shit

**Comment by lucidillusions** [comment]
love it ! 

can we have some different icons/shades for like/repost in kawaii mode please?

**Pull request merged**